### PR TITLE
Added domain_shopping option.  Added additional fields for domain_org…

### PR DESCRIPTION
**Pull request by michaelaguiar**
Added domain_shopping option.  Added additional fields for domain_organic and domain_adwords

### CODE REVIEW

**Comment by michaelaguiar** [comment]
Looks like it's failing for not finding the GuzzleHttp/Client library -- which is external via Composer, so normal issue?

File: src/Client.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 use Silktide\SemRushApi\Model\Factory\ResultFactory;
 use Silktide\SemRushApi\Model\Request;
 use Silktide\SemRushApi\Model\Result as ApiResult;
-use Guzzle\Http\Client as GuzzleClient;
+use GuzzleHttp\Client as GuzzleClient;
```
Comment (andywaite) [inline src/Client.php:13]: Why was this changed?  Seems to have broken the build.

Comment (dochne) [inline src/Client.php:13]: If I recall correctly, the difference between GuzzleHttp and Guzzle\Http is a breaking change between Guzzle v3 and v4? @andywaite 

GuzzleHttp is what we should be using, you're using Guzzle which is being pulled in as a dev dependancy from php-coveralls. Technically, the code as is isn't working. If you pull down off your non-dev dependancies, it won't work

Issue #7
